lib/filepicker_rails/policy.rb

```diff
@@ -19,9 +19,9 @@ def signature
       OpenSSL::HMAC.hexdigest('sha256', ::Rails.application.config.filepicker_rails.secret_key, policy)
     end
 
-    def self.apply(call = [:read, :convert], keys = ['policy', 'signature'])
+    def self.apply(call: [:read, :convert], keys: ['policy', 'signature'], options: {})
```

Comment:
A change to keyword arguments would break support for 1.9.3, however 1.9.3 has been failing master for months #155